### ISSUE TITLE
Docs should begin with TOC

### ISSUE THREAD

**Issue by SetTrend** [less/less-docs#361]
It's quite annoying to be required to scroll through the whole document for finding information about a topic.

As a reference document, the LESS documentation (*Language Features* and *Function Reference*) should begin with a hyperlinked TOC containing all H1/H2/H3 headers.

**Comment by seven-phases-max** [comment]
Well, the H1 TOC at the left navbar (also contaning H2 subitems). Technically it's neither a book nor a reference document (but more like a bunch of tutorials + a few distinct reference topics like "Functions"), so a TOC would be too chaotic and sort of useless (about 3-5 screens of pretty random stuff), imho. Maybe an index or sort of would make more sense.

**Comment by SetTrend** [comment]
That's strange. When I open the doc's I don't see any TOC or index:

![toc](https://cloud.githubusercontent.com/assets/9283914/9810433/09552cc4-5871-11e5-8e74-22cab5ff0cf3.png)

TOC or index, it's all the same: Just a nested list of H1/H2/H3 elements, reflecting their contents. That would help much in using LESS.

**Comment by lukeapage** [comment]
what browser is that? what happens if the page is wider? looks like something responsive hides it..

**Comment by seven-phases-max** [comment]
>TOC or index, it's all the same

No, by index I mean something like [this](http://www.w3.org/TR/css3-values/#index).
And yes, what it's in the screenshot is an issue actually, on narrow width screens the navbar is hidden (probably ideally we should make it accessible there with another dropdown button).

**Comment by SetTrend** [comment]
Aahhh.... sooo.... OK, now I see... YEEESSS ... That's what I've desperately been looking for.

Excellent!

Thanks for pointing me.

May I suggest to plain put it above content on smaller screens? The TOC is often the first place to look for information.

**Comment by matthew-dean** [comment]
We should do a mobile-style left-hand menu, where the page slides out of the way revealing the navigation. Stacking it above content will be too heavy.
